packages/react/transform/turbo.json

```diff
@@ -1,19 +0,0 @@
-{
-  "$schema": "https://turbo.build/schema.json",
-  "extends": ["//"],
-  "tasks": {
-    "build:debug": {
-      "inputs": ["src/**/*.rs", "build.rs", "Cargo.toml"],
-      "outputs": ["index.d.ts", "index.cjs", "*.node"]
-    },
-    "build:wasm": {
-      "inputs": ["src/**/*.rs", "src/wasm.js", "build.rs", "Cargo.toml", "scripts/build_wasm.sh"],
-      "outputs": ["dist/wasm.cjs"]
-    },
-    "build": {
```

Comment:
Maybe we could keep this task and make it dependsOn `@lynx-js/swc-plugin-reactlynx#build` and `@lynx-js/swc-plugin-compat`.